src/examples/bad_code.cpp

```diff
@@ -0,0 +1,49 @@
+#include "ros/ros.h"
+#include "std_msgs/Float64.h"
+
+ros::Publisher pub;
+//pretend I commented everything
+double calc_pi    (void)
```

Comment:
Too much space after the function name. Either use one space or no space after the name.